client/src/services/stakeServices.js

```diff
@@ -48,6 +43,6 @@ export const getSignedStakeDeploy = async ({
 		const signedDeploy = await signDeploy(deploy, fromAddress, validator);
 		return signedDeploy;
 	} catch (error) {
-		throw error;
+		throw new Error(`Failed to get signed stake deploy due to ${error}`);
```

Comment:
hi @djhahe , as your explanation about no-useless-catch, I did the changes by throwing new error exception with helpful message. It might help us to support customer better.

Reply:
thanks, you can merge the PR